File: CalculadoraDeImc/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CalculadoraDeImc
{
    public class Pessoa
    {
        public string name { get; set; }
        public int age { get; set; }
        public double weight { get; set; }
        public double height { get; set; }


        public Pessoa()
        {
        }
        public Pessoa(string name, int age, double weight, double height)
        {
            this.name = name;
            this.age = age;
            this.weight = weight;
            this.height = height;
        }
        public double IMC()
        {
            return weight / (height * height);
        }

        public string IMCtxt()
        {
            double imc = IMC();
            if (imc < 0.0018)
                return "Abaixo do peso";
            else if (imc < 0.0025)
                return "Peso normal";
            else if (imc < 0.0030)
                return "Sobrepeso";
            else
                return "Obesidade";
        }


    }
}

File: CalculadoraIMCJV/src/Pessoa.java
public class Pessoa {
    protected String name;
    protected int age;
    protected double weight;
    protected double height;

    public Pessoa (){

    }

    public Pessoa(String name, int age, double weight, double height) {
        this.name = name;
        this.age = age;
        this.weight = weight;
        this.height = height;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public double getWeight() {
        return weight;
    }

    public void setWeight(double weight) {
        this.weight = weight;
    }

    public double getHeight() {
        return height;
    }

    public void setHeight(double height) {
        this.height = height;
    }
    public double IMC(){
        return weight / (height * height);
    }
    public String IMCtxt() {
        double imc = IMC();
        if (imc < 18) return "Abaixo do peso";
        else if (imc < 25) return "Peso normal";
        else if (imc < 30) return "Sobrepeso";
        else return "Obesidade";
    }
}
